R/model.R

```diff
@@ -517,3 +526,75 @@ def run(input_data):
   close(py_file)
   invisible(NULL)
 }
+
+#' Register a model for operationalization.
+#'
+#' @description
+#' Register a model for operationalization.
+#'
+#' @param run The `Run` object.
+#' @param model_name The name of the model.
+#' @param model_path The relative cloud path to the model, for example,
+#' "outputs/modelname". When not specified, `model_name` is used as the path.
+#' @param tags A dictionary of key value tags to assign to the model.
+#' @param properties A dictionary of key value properties to assign to the model.
+#' These properties cannot be changed after model creation, however new key-value pairs can be added.
+#' @param model_framework Framework of the model to register. Currently supported
```

Comment:
we should remove the `model_framework` and `model_framework_version` parameters since they won't apply here.

Reply:
got it